Start of project1/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <figure>
+        <img src="images/octopus092118.jpg" alt="Lalala">
+        <figcation>
+
+        </figcation>
+    </figure>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why? </h1>
+    <p>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</p>
+
+
+    <p><img src="https://static01.nyt.com/images/2018/12/04/science/04SCI-ZIMMER/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg?quality=90&auto=web"
```

Comment:
A few notes here:
1. You can delete the opening `p` tag here -- an `img` can stand on its own and doesn't need to be wrapped in a `p` element.
1. Remember to add an `alt` attribute to all your images. We'll talk more about this in class so you'll know what makes a good `alt` text and why it's important.

```suggestion
    <img src="https://static01.nyt.com/images/2018/12/04/science/04SCI-ZIMMER/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg?quality=90&auto=web" alt=""
```

3. Remember that you can't link to images on other sites for your final project; you'll want to download this image and put it in the `images` directory in your project folder.